average.js

```diff
@@ -1,6 +1,13 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  // return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
```

Comment:
Don't leave commented code in commits... if we want to see old code, we can always see it in GitHub history :)